spotify-conversion-next/pages/index.tsx

```diff
@@ -13,83 +13,50 @@ import { NextPage } from "next";
 import Link from "next/link";
 import login from "./api/login";
 import spotifyQueries from "@/services/spotify/spotifyQueries";
+import spotifyRoutes from "@/services/spotify/spotifyRoutes";
+import { useSession } from "next-auth/react";
 
 type SpotifyFormFields = {
   playlistLink: string;
   playlistName: string;
 };
 
-// TODO: Add these to a .env.local file
-const SPOTIFY_REDIRECT_URI = "http://localhost:3000/spotify-callback/";
-const SPOTIFY_SECRET = "5ff7e5b8b9c24237afed36bece1af9fc";
-const SPOTIFY_ID = "b1263098ab4b4da7911e4c3c55f6621e";
-
-// export async function getServerSideProps() {
-//   const res = await fetch(
-//     "https://api.spotify.com/v1/tracks/11dFghVXANMlKmJXsNCbNl"
-
-//   );
-//   const data = await res.json();
-
-//   return {
-//     props: { data },
-//   };
-// }
+export async function getServerSideProps() {
+  return {
+    props: {},
+  };
+}
 
 type HomeProps = {};
 const Home: NextPage<HomeProps> = () => {
   const { register, handleSubmit } = useForm();
+  const { data: session } = useSession();
```

Comment:
Here's that `useSession()` react hook. Now, from our FE, we have the `session` variable, and we can get data related to our Session, such as the User's username, email, profile image, etc.